flask_googlemaps/__init__.py

```diff
@@ -514,10 +516,14 @@ def add_polyline(self, path=None, **kwargs):
             if not isinstance(path, list):
                 raise AttributeError('The path is a list of dictionary of'
                                      'latitude and longitudes por path points')
-            for point in path:
+            for i in range(len(path)):
+                point = path[i]
                 if not isinstance(point, dict):
```

Comment:
Yes, enumerate is better for this case. "i" is required to replace an element (list, tuple) of path list by dictionary with lat and lng. This is done to make such polygon/polyline definition possible. 
`{
            'stroke_color': '#0AB0DE',
            'stroke_opacity': 1.0,
            'stroke_weight': 3,
            'path': [
                [59.94, 30.318],
                [59.946, 30.325],
                [59.946, 30.34],
                [59.941, 30.35],
                [59.938, 30.33]
            ],
            'infobox': 'This is a polygon'
        }`